deploy/crds/enterprise.splunk.com_clustermasters_crd.yaml

```diff
@@ -660,17 +660,27 @@ spec:
                 by commas
               type: string
             defaultsUrlApps:
-              description: Full path or URL for one or more default.yml files for installing apps, separated
-                by commas
-              type: string
-            ephemeralStorage:
-              description: If true, ephemeral (emptyDir) storage will be used for
-                /opt/splunk/etc and /opt/splunk/var volumes
-              type: boolean
-            etcStorage:
-              description: Storage capacity to request for /opt/splunk/etc persistent
-                volume claims (default=”1Gi”)
+              description: Full path or URL for one or more default.yml files specific
+                to App install, separated by commas This is meant as a temporary fix
```

Comment:
That works with me.  So this would be something like:
```
              description: Full path or URL for one or more default.yml files specific
                to App install, separated by commas.  The defaults listed here
                will be installed on the CM instance.
```